frontend/index.html

```diff
@@ -151,65 +157,78 @@ <h2><a class="hash-link" href="#continued-requirements">Continued Requirements</
-        </ul>
-      </li>
-      <li>Add the <code>Strict-Transport-Security</code> header to all HTTPS responses and ramp up the <code>max-age</code> in stages, using the following header values:
-        <ul>
-          <li>
-            5 minutes:<br>
-            <code class="header-example">max-age=300; includeSubDomains</code>
-          </li>
-          <li>
-            1 week:<br>
-            <code class="header-example">max-age=604800; includeSubDomains</code>
-          </li>
-          <li>
-            1 month:<br>
-            <code class="header-example">max-age=2592000; includeSubDomains</code>
-          </li>
-        </ul>
-        During each stage, check for broken pages and monitor your site's metrics (e.g. traffic, revenue). Fix any problems that come up and then wait the full <code>max-age</code> of the stage before you move on. For example, wait a month in the last stage.
-      <li>Once you're confident that there will be no more issues, increase the <code>max-age</code> to 2 years and submit your site to the preload list:
-        <ul>
-          <li>
-            2 years, requesting to be preloaded:<br>
-            <code class="header-example">max-age=63072000; includeSubDomains; preload</code>
-          </li>
-        </ul>
-      </li>
-    </ol>
-
     <p>
-      If you have a group of employees or users who can beta test the deployment, consider trying the first few ramp-up stages on those users. Then make sure to go through all stages for all users, starting over from the beginning.
+      Projects that support or advise about HSTS and HSTS preloading should ensure that site operators understand the long-term consequences of preloading before they turn it on for a given domain. They should also be informed that they need to meet additional requirements and submit their site to <a href="https://hstspreload.org/">hstspreload.org</a> to ensure that it is successfully preloaded (i.e. to get the full protection of the intended configuration).
     </p>
+  </section>
+</div>
 
+<div class="content form">
+  <section id="submission-form" class="form">
+    <h2><a class="hash-link" href="#submission-form">Submission Form</a></h2>
     <p>
-      Consult the <a href="https://wiki.mozilla.org/Security/Guidelines/Web_Security">Mozilla Web Security guidelines</a> and the <a href="https://developers.google.com/web/fundamentals/security/?hl=en">Google Web Fundamentals pages on security</a> for more concrete advice about HTTPS deployment.
+      If you still wish to submit your domain for inclusion in Chrome's HSTS preload list and you have followed our <a href="#deployment-recommendations">deployment recommendations</a> of slowly ramping up the <code>max-age</code> of your site's <code>Strict-Transport-Security</code> header, you can use this form to do so:
     </p>
 
+    <form id="domain-form" action="#submission-form" class="hidden">
+      <p>
+        <label for="domain">Domain to preload:</label>
```

Comment:
Teensy nit, but this throws off the centering, something like this would easily keep it centered:
```suggestion
        <hr>
        <label for="domain">Domain to preload:</label>
        <br>
```